Dockerfile

```diff
@@ -16,11 +16,7 @@ RUN apt-get update && apt-get install -y cloudflared
 COPY ./ssh-client.conf /root/ssh-client.conf
 RUN cat /root/.ssh/config
 COPY ./entrypoint.sh /root/entrypoint.sh
-COPY ./expect.sh /root/expect.sh
```

Comment:
With the change to using `ssh -v -o "StrictHostKeyChecking no" -p $PORT $USER@$HOST "$REMOTE_SHELL" < commands.txt`, there was no more need for the expect file that runs an interactive SSH session